lib/widgets/post/post.dart

```diff
@@ -58,4 +69,24 @@ class OBPost extends StatelessWidget {
       ],
     );
   }
+
+  void _bootstrap(BuildContext context) {
+    InViewState _inViewState;
+    if (inViewId != null) {
+      _inViewState = InViewNotifierList.of(context);
+      _inViewState.addContext(context: context, id: inViewId);
```

Comment:
This will mess up the video autoplaying as videos will be played as soon as another post enters into the view as some fill the entire thing, instead of the video part. Results in premature playing of videos.

Use a different inViewId for this detection.

Reply:
Oh i misunderstood how it works, got it now !

Reply:
Alright. You will push new changes right? I can't see any more commits after these were marked as resolved.